localization/english/event_localization/wc_arthas_expedition_events_l_english.yml

```diff
@@ -86,12 +86,12 @@
  wc_arthas_story_55_DESC_arthas_muradin:0 "[muradin.GetFirstName] [muradin.GetDynastyHouseName], my old mentor and friend.\n\n"
  wc_arthas_story_55_DESC_arthas_not_muradin:0 "[muradin.GetFirstName] [muradin.GetDynastyHouseName].\n\n"
  wc_arthas_story_55_DESC_arthas_ending:0 "After establishing our base camp, [muradin.GetSheHe] said that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne. Could it be a similar blade like the one that has haunted my dreams?"
- wc_arthas_story_55_DESC_group_beggining:0 "After a month of sailing, we have arrived in Northrend. The howling wind cuts to the bone.\n\nWe didn't travel far inland until we found a [muradin.GetCulture.GetGroup.GetName] camp and their leader [muradin.GetFirstName] [muradin.GetDynastyHouseName],"
- wc_arthas_story_55_DESC_group_muradin:0 ", [arthas.GetTitledFirstNamePossessive] old teacher and friend."
- wc_arthas_story_55_DESC_group_ending:0 "[muradin.GetSheHe|U] says that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne.\n\nWe joined up with [muradin.GetFirstNamePossessive] encampment and are now prepared to battle the undead that roam these frozen lands. Some of [muradin.GetFirstNamePossessive] forces have been cut off from [muradin.GetFirstName] camp. Our expedition plans to rescue these riflemen and face the undead head on."
+ wc_arthas_story_55_DESC_group_beggining:0 "After a month of sailing, we have arrived in Northrend. The howling wind cuts to the bone.\n\nWe didn't travel far inland until we found a [muradin.GetCulture.GetGroup.GetName] camp and their leader"
+ wc_arthas_story_55_DESC_group_muradin:0 "[muradin.GetFirstName] [muradin.GetDynastyHouseName], [arthas.Custom2('GetTitledFirstNamePossessiveOrMy', ROOT.Char)] old teacher and friend."
+ wc_arthas_story_55_DESC_group_ending:0 "[muradin.GetSheHe|U] says that [muradin.GetHerHis] purpose here is to recover a runeblade called Frostmourne.\n\nWe joined up with [muradin.GetFirstNamePossessive] encampment and are now prepared to battle the undead that roam these frozen lands. Some of [muradin.Custom2('GetFirstNamePossessiveOrMy', ROOT.Char)] forces have been cut off from [muradin.Custom2('GetFirstNamePossessiveOrMy', ROOT.Char)] camp. Our expedition plans to rescue these riflemen and face the undead head on."
```

Comment:
This change doesn't make sense, why would some of the forces of an expedition member be cut off?

Reply:
> This change doesn't make sense, why would some of the forces of an expedition member be cut off?

Muradin? He had an dwarven expedition looking for Frostmourne in WC3.

Reply:
But why have it "GetFirstNamePossessiveOrMy" then? My would target the group member, wouldn't it? So a group member would read "Some of my forces have been cut off my camp.", right?

Reply:
> But why have it "GetFirstNamePossessiveOrMy" then? My would target the group member, wouldn't it? So a group member would read "Some of my forces have been cut off my camp.", right?

No, it should be...

> Some of his forces

Reply:
As oh, if that's how it's defined, i have no objections.